src/base/CMakeLists.txt

```diff
@@ -75,3 +78,14 @@ set_source_files_properties(
   ${CMAKE_CURRENT_BINARY_DIR}/Trace_tags.h
   PROPERTIES GENERATED TRUE
 )
+
+add_library(cvc4base STATIC ${LIBBASE_SOURCES})
+if(ENABLE_SHARED)
+  set_target_properties(cvc4base PROPERTIES POSITION_INDEPENDENT_CODE ON)
+endif()
+target_compile_definitions(cvc4base
+  PRIVATE
+    -D__BUILDING_CVC4LIB
+    -D__STDC_LIMIT_MACROS
+    -D__STDC_FORMAT_MACROS
```

Comment:
What do these do?

Reply:
That's what I've been wondering too. It is an old relic from the old build infrastructure, maybe we can get rid of it.

Reply:
I removed it.